### PR TITLE
Fix JSON RPC example for Parcel changes

### PR DESCRIPTION
**Pull request by gurrpi**
Parcel's r, s, v field replaced by sig field by https://github.com/CodeChain-io/codechain/commit/0990b704e4d790816e6b9195c4271eb905ebbabd#diff-501d66daa8d28a1195a160679e4b47ff

### CODE REVIEW

**Comment by gurrpi** [comment]
Done.

File: spec/JSON-RPC.md [hunk 1]
```diff
@@ -258,52 +258,52 @@ Request Example:
...
+    "invoicesRoot":"0x3a14d04383882243a684a6b0e779905f7883b12b5fb3ebf738facfcd2095b77a",
+    "number":5,
     "parcels":[
       {
         "action":{
-          "action":"changeShardState",
+          "action":"payment",
+          "amount":"0xa",
+          "receiver":0xa6594b7196808d161b6fb137e781abbc251385d9
           "transactions":[
```
Comment (joojis) [inline spec/JSON-RPC.md]: The transaction field belongs to changeShardState. Please remove this field

**Comment by joojis** [comment]
@gurrpi Thank you for your patch. I added 1 comment. Could you check?

**Comment by gurrpi** [comment]
@joojis I fixed it! Thank you for inform me.

**Pull request merged**